docs/config.md

```diff
@@ -1919,10 +1919,9 @@ Each field in this section is a `flag`.
 
 #### `Swarm.Transports.Network.TCP`
 
-[TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) is the most
-widely used transport by Kubo nodes. It doesn't directly support encryption
-and/or multiplexing, so libp2p will layer a security & multiplexing transport
-over it.
+[TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) doesn't
+directly support encryption and/or multiplexing, so libp2p will layer a security
+& multiplexing transport over it.
```

Comment:
```suggestion
[TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) is a simple
and well deployed transport, it should be inter-compatible with most
implementations and network configurations.
TCP doesn't directly support encryption and/or multiplexing, so libp2p will
layer a security & multiplexing transport over it.
```

Reply:
"Widely deployed" instead of "well deployed". "should be compatible" instead of "inter-compatible".

besides that, looks good.

Reply:
:+1: will you commit then I merge ?

Reply:
I meant well deployed as:
> Should be resistant-ish to middle boxes fucking up things they don't know about.

(because they most likely know about TCP)
Yours sounds better.